Decimus/Lib/libquicr/QCommon.h

```diff
@@ -36,8 +36,10 @@ typedef struct QObjectHeaders {
 
 #ifdef __cplusplus
 static quicr::FullTrackName ftnConvert(QFullTrackName qFtn) {
+    const auto nameSpace = std::vector<std::uint8_t>(qFtn.nameSpace, qFtn.nameSpace + qFtn.nameSpaceLength);
+    std::vector<std::vector<std::uint8_t>> tuple = { nameSpace };
     return {
-        .name_space = std::vector<std::uint8_t>(qFtn.nameSpace, qFtn.nameSpace + qFtn.nameSpaceLength),
+        .name_space = quicr::TrackNamespace(tuple),
```

Comment:
Don't need to store this, could just do it inline

Reply:
This is actually a copy, so probably discourage this since copying the vector isn't needed.